### PR TITLE
[WIP] Add series nav

### PR DESCRIPTION
**Pull request by johnharris85**
This PR adds the series concept and initial two series.

Still WIP:

- [ ] Tidy Series overview pages
- [ ] Decide on name, series? outcome?
- [ ] Styling / UX review
- [ ] Prev / next navigation (in progress @zmb3 )
- [ ] Add related content boxes (in progress @zmb3)
- [ ] Fix tree nav displays
- [ ] Handle external links in template
- [ ] Filter out source meta field from spell check?
- [ ] Add README for adding series content

### CODE REVIEW

**Comment by zmb3** [comment]
Is there something unique about the files under the theme's `layouts/series` directory, or are these just copy pasted from other places? If so, we've got separate issues to try and reduce that duplication, so it would be nice if this PR didn't introduce yet another copy of the same files 😄

**Comment by johnharris85** [comment]
> Is there something unique about the files under the theme's `layouts/series` directory, or are these just copy pasted from other places? If so, we've got separate issues to try and reduce that duplication, so it would be nice if this PR didn't introduce yet another copy of the same files 
> 
> Also, if a simple `hugo server` command no longer fully renders the site because some additional content has to be auto-generated, I would expect to see the steps for doing so called out in the README.

They're unique. And `hugo server` should always render fine (if you're not messing with series at all). If you _are_ creating / editing series then yes absolutely, adding steps to the readme is called out as an action above.

File: content/series/building-k8s-app-platform/_index.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+          source: "/guides/kubernetes/workload-tenancy-cluster-tuning"
+          weight: 4
+        - title: Pod Priority and Preemption
+          source: "/guides/kubernetes/workload-tenancy-priority-preemption"
+          weight: 5
+---
+
+## Building a Kubernetes Application Platform
+
+Kubernetes is a primitive for building platforms at most organizations. Some organizations prefer a turn-key solution such as PAS that can abstract the underlying orchestrator, while others prefer to compose their platform based on cloud native technologies that fit their business needs. The CNFE team's customers typically fall in the latter category. For our customers, this journey includes the following concerns.
```
Comment (zmb3) [inline content/series/building-k8s-app-platform/_index.md:92]: - Replace PAS with Tanzu Application Service
- Remove references to CNFE
- Remove reference to Pacific
- "VMware's Tanzu TKG" stutters
- Remove reference to CNR?

**Comment by bryanfriedman** [comment]
Note, this PR was superceded by https://github.com/vmware-tanzu/tanzu-dev-portal/pull/698/files but keeping the branch around for a bit so we can potentially pull the automated Python pieces from it...

**Comment by BrianMMcClain** [comment]
@johnharris85 @zmb3 @bryanfriedman any thoughts on what you'd like to do with this PR? I know we had implemented this in another manner, but we had talked about borrowing some of the code from this in the future. Didn't want to just close without reaching out.

**Comment by johnharris85** [comment]
Ye let's close. I have the code still locally if we need it.

**Pull request closed without merging**